ITT fee increase (#4361)
DFE-Digital/get-into-teaching-app: app/views/content/funding-and-support/tuition-fees.md

      </p>
      <p>
      Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.</p>
inset_text:
  fee-increase:
    text: |-

>       From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses have increased. The increased amounts are shown on this page. If you've already applied to a course, speak with your provider to check if they will be increasing their fees.

Reviewer: 'For a course'? Rather than 'to a course'?

>       From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses have increased. The increased amounts are shown on this page. If you've already applied to a course, speak with your provider to check if they will be increasing their fees.

Reviewer: ```suggestion
      From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses have increased. The increased amounts are shown on this page. If you've already applied to a course, speak with your provider to check if they'll be increasing their fees.
```

>       From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses have increased. The increased amounts are shown on this page. If you've already applied to a course, speak with your provider to check if they will be increasing their fees.

Author: ```suggestion
      From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses have increased. The increased amounts are shown on this page. If you've already applied for a course, speak with your provider to check if they will be increasing their fees.
```

      </p>
      <p>
      Find out more about <a href="/non-uk-teachers/fees-and-funding-for-non-uk-trainees">fees and financial support for non-UK trainee teachers</a>.</p>
inset_text:
  fee-increase:
    text: |-

>       In autum 2025, the maximum fees that providers can charge for postgraduate teacher training courses increased. These are the fees shown on this page. 

Reviewer 2: ```suggestion
      In autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses increased. These are the fees shown on this page. 
```

Reviewer 2: @emmaachesongray I don't think this sentence quite makes sense - do you mean `will increase` / `have been increased` / something similar?

>       In autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses increased. These are the fees shown on this page. 

Author: ```suggestion
      Since autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses has been increased. The increased amounts are shown on this page. 
```

Author: @gemmadallmandfe is this clearer?

Reviewer 2: Would `From autumn 2025, the maximum fees that providers can charge for postgraduate teacher training courses has been increased. The increased amounts are shown on this page.` be slightly better?
